tfhe/src/core_crypto/entities/lwe_multi_bit_bootstrap_key.rs

```diff
@@ -284,7 +283,8 @@ impl<Scalar: UnsignedInteger> LweMultiBitBootstrapKeyOwned<Scalar> {
     }
 }
 
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+#[serde(bound(deserialize = "C: IntoContainerOwned"))]
```

Comment:
the reason for the serde bounds is because we don't use the default deserialize implementation, but instead permute the elements since values in the fourier domain are unordered